tests/cypress/e2e/archiving-assets.cy.ts

```diff
@@ -0,0 +1,27 @@
+import { testCases } from './../../test-cases';
+
+// TODO: Remove when Cypress support achieves parity with Playwright
+const skippedTestCases = [
+  'Asset Paths / asset doesnt prevent directory from being created',
+  'Asset Paths / percents in URLs are handled',
+  'Asset Paths / external asset is archived',
+  'Asset Paths / assets from css urls are archived',
+];
```

Comment:
Not ideal, but Cypress (this is still using `cy.intercept` in this branch) doesn't handle these cases. We should be able to get rid of these when Cypress moves to CDP in a soon PR.